gradle.properties

```
@@ -1,5 +1,5 @@
 GROUP=com.afterpay
-VERSION_NAME=4.5.1-SNAPSHOT
+VERSION_NAME=4.6.0
```

We just keep the `-SNAPSHOT` section here. Github actions removes it when deploying. See [here](https://github.com/afterpay/sdk-android/blob/v4.5.1/.github/workflows/release.yml#L21)

```suggestion
VERSION_NAME=4.6.0-SNAPSHOT
```